CMakeLists.txt

```diff
@@ -1,5 +1,8 @@
 cmake_minimum_required(VERSION 3.1)
-project(apriltag VERSION 3.3.0 LANGUAGES C CXX)
+project(apriltag
+        VERSION 3.3.0
```

Comment:
While we are it - does it make sense to read the version from the package.xml such that we don't maintain the numbering in two places?

Reply:
Is this possible? I know you can run custom commands in CMake, but I've not seen a project that parses the xml to extract the `version` tag. If this is possible with onboard tools (i.e. without installing additional software) then this would be an improvement to the error-prone manual version updates.

I've seen CMake examples getting the version from the git tag. But in any case, I don't think there is a way to set the `package.xml` version from either git or CMake.